src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -24,12 +25,15 @@ const strings = {
 export default class FiatConnectQuote extends NormalizedQuote {
   quote: FiatConnectQuoteSuccess
   fiatAccountType: FiatAccountType
+  flow: CICOFlow
   constructor({
     quote,
     fiatAccountType,
+    flow,
   }: {
     quote: FiatConnectQuoteSuccess
     fiatAccountType: keyof typeof quote.fiatAccount
+    flow: CICOFlow
```

Comment:
it does seem kinda weird that quotes don't have info on whether they're for in or out. since both the client and the server will care about that, and will presumably be tracking that separately

Reply:
I added a comment to update once this spec change is pushed through: https://github.com/fiatconnect/specification/pull/67

Reply:
ty